swap/src/cli/command.rs

```diff
@@ -15,6 +15,9 @@ const DEFAULT_ELECTRUM_RPC_URL: &str = "ssl://electrum.blockstream.info:60002";
 
 const DEFAULT_TOR_SOCKS5_PORT: &str = "9050";
 
+// Bitcoin transactions should be confirmed within X blocks
+const DEFAUL_BITCOIN_CONFIRMATION_TARGET: &str = "3";
```

Comment:
```suggestion
const DEFAULT_BITCOIN_CONFIRMATION_TARGET: &str = "3";
```

missed that one before, can do in follow up cleanup ;)

Reply:
I'll fix it quickly.